### DISCUSSION TITLE
How to make Tooltip's Arrow to always point to Trigger's center?

### DISCUSSION THREAD

**Discussion by moroshko** [radix-ui/primitives#1258] [Help]
When the trigger is positioned in the middle of the screen, Tooltip's arrow points by default to the Trigger's center:
<img width="169" alt="Screen Shot 2022-03-19 at 1 08 57 pm" src="https://user-images.githubusercontent.com/259753/159102823-181e8295-15b7-4c01-900b-bc8a0c82d9cb.png">

But, when the trigger is positioned at the edge of the viewport, the arrow is not pointing to the button's center anymore:
<img width="142" alt="Screen Shot 2022-03-19 at 1 09 17 pm" src="https://user-images.githubusercontent.com/259753/159102863-ae5f607a-df3d-403b-a456-051fb9f34209.png">

Is there a way to configure Tooltip's Arrow to **always** point to the Trigger's center like below without manually calculating arrow's offset?
<img width="142" alt="Screen Shot 2022-03-19 at 1 10 30 pm" src="https://user-images.githubusercontent.com/259753/159102899-5b709182-9369-4fd8-b1b0-5d748ac40a55.png">

[CodeSandbox](https://codesandbox.io/s/radix-tooltip-arrow-position-1vo1y0)

**Comment by moroshko** [comment]
@jjenzz Curious to hear your thoughts 🙏

**Reply by jjenzz** [reply]
i'm not sure this is possible at the moment without doing something like you describe. it does seem to make sense to have the arrow respect the requested positioning though. feature request perhaps?

**Reply by benoitgrelard** [reply]
That's correct, currently `Arrow` `offset` prop is the only way to achieve this.
From memory we chose this approach as it allowed us to avoid lots of edge cases with the other suggested approach.
But I agree, feel free to open a feature request!
